frappe/utils/__init__.py

```diff
@@ -161,6 +161,21 @@ def split_emails(txt):
 
 	return email_list
 
+def validate_url(txt, throw=False):
+	try:
+		url = urlparse(txt).netloc
+		if not url:
+			raise frappe.ValidationError
+	except Exception:
+		if throw:
+			frappe.throw(
+				frappe._(
+					"'{0}' is not a valid URL"
+				).format('<strong>' + txt +'</strong>')
```

Comment:
Translation strings need to be in one line to be scannable.

https://frappeframework.com/docs/user/en/translations

Reply:
fixed semgrep check for line break
https://github.com/frappe/frappe/pull/12876/commits/edfe4adb3fafcfb388c450d46ff6ffab15e709f5

Reply:
👍 Fixed. @netchampfaris